Add minimal Sphinx for ReadTheDocs

nottombrown opened the pull request:
![image](https://user-images.githubusercontent.com/306655/30672333-b122f084-9e20-11e7-87ce-5a4534ce30b4.png)

goodfeli commented on docs/conf.py:
```diff
@@ -0,0 +1,175 @@
+
+# General information about the project.
+project = 'CleverHans'
+copyright = '2017, Ian Goodfellow, Nicolas Papernot, Ryan Sheatsley'
```
Can we delete this line?

goodfeli replied:
(the line about "copyright")

goodfeli commented on docs/conf.py:
```diff
@@ -0,0 +1,175 @@
+# built documents.
+#
+# The short X.Y version.
+version = '2.0.0'
```
Can we get this to say '-dev' when it's not run directly on the tagged release?

goodfeli commented:
Made two small comments. Looks great, thanks!

nottombrown commented:
Removed version number and added better introductory text. Up to date version here:
![image](https://user-images.githubusercontent.com/306655/30672792-6fbae00e-9e23-11e7-8212-a819b71fb4ee.png)

goodfeli commented:
LGTM. Feel free to merge once the tests pass
